xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-webjar/src/main/webjar/searchSuggest.js

```diff
@@ -131,10 +140,9 @@ var XWiki = (function (XWiki) {
             .insert(new Element('span', {'class':'suggestValue'}))
             .insert(new Element('span', {'class':'suggestInfo'}));
       this.noResultsMessage = new Element('div', {'class': 'hidden'})
-        .update("$escapetool.javascript($services.localization.render('core.widgets.suggest.noResults'))".escapeHTML());
-      var gotoSearchPageMessage = new Element('div')
-        .update("$escapetool.javascript($services.localization.render('core.widgets.suggest.showResults'))"
-        .escapeHTML());
+        .update(this._l10n['core.widgets.suggest.noResults']).escapeHTML();
+      var gotoSearchPageMessage = new Element('div').update(this._l10n['core.widgets.suggest.showResults'])
+        .escapeHTML();
```

Comment:
Same as above, I don't understand how this works since ``update`` returns an ``Element`` while http://api.prototypejs.org/language/String/prototype/escapeHTML/index.html is a ``String`` function.